docs/configuration.md

```diff
@@ -25,69 +27,75 @@ Logger:
 This section lists configurations that may affect the performance.
 
 | Name              | Description                                                  | Range             | Default |
 | ----------------- | ------------------------------------------------------------ | ----------------- | ------- |
-| entryFormat       | The format of an entry. Changing it to pulsar will avoid unnecessary encoding and decoding to improve the performance, whose cost is that a topic cannot be used by mixed Pulsar clients and Kafka clients. | kafka,<br> pulsar | kafka   |
-| maxReadEntriesNum | Maximum number of entries that are read from cursor once per time.<br>Increasing this value can make FETCH request read more bytes each time.<br>NOTE: KoP doesn't check the max bytes limit currently so if the value is too large, the response size may be over the network limit. |                   | 5       |
+| entryFormat       | The format of an entry. Changing it to `pulsar` will avoid unnecessary encoding and decoding to improve the performance, whose cost is that a topic cannot be used by mixed Pulsar clients and Kafka clients. | kafka,<br> pulsar | kafka   |
+| maxReadEntriesNum | The maximum number of entries that are read from the cursor once per time.<br>Increasing this value can make FETCH request read more bytes each time.<br>**NOTE**: Currently, KoP does not check the maximum byte limit. Therefore, if the value is too great, the response size may be over the network limit. |                   | 5       |
 
 ## Network
 
 | Name              | Description                                                  | Default |
 | ----------------- | ------------------------------------------------------------ | ------- |
-| maxQueuedRequests | Limit the queue size for request, like queued.max.requests in kafka server. | 500     |
-| requestTimeoutMs  | Limit the timeout in milliseconds for request, like request.timeout.ms in kafka client.<br>If a request was not processed in the timeout, KoP would return an error response to client. | 30000   |
+| maxQueuedRequests | Limit the queue size for request, like `queued.max.requests` in Kafka server. | 500     |
+| requestTimeoutMs  | Limit the timeout in milliseconds for request, like `request.timeout.ms` in Kafka client.<br>If a request was not processed in the timeout, KoP would return an error response to client. | 30000   |
 
-> NOTE: these limits are based on each connection.
+> **NOTE**
+> 
+> These limits are based on each connection.
 
 ## Prometheus
 
 | Name                                     | Description                                                  | Default |
 | ---------------------------------------- | ------------------------------------------------------------ | ------- |
-| kopPrometheusStatsLatencyRolloverSeconds | Kop metrics expose to prometheus rollover latency in seconds. | 60      |
+| kopPrometheusStatsLatencyRolloverSeconds | Kop metrics exposed toPprometheus rollover latency in seconds. | 60      |
```

Comment:
```suggestion
| kopPrometheusStatsLatencyRolloverSeconds | Kop metrics exposed to Prometheus rollover latency in seconds. | 60      |
```

Did you just only want to change `expose` to `exposed`? I think `prometheus` or `Prometheus` is both okay.

Reply:
Updated to exposed to prometheus rollover....PTAL